README.md

```diff
@@ -1,60 +1,35 @@
-# Thorntail Booster License Generator
+# Thorntail Examples License Generator
 
 Example usage:
 
 ```bash
-$ mvn clean package -Dbooster.project.dir=/path/to/thorntail-rest-http -Dbooster.name="Thorntail - REST API Level 0" -Dbooster.version=2.4.0-redhat-1 -Dbooster.assembly.name=thorntail-rest-http -Dbooster.product.build=true
+$ mvn clean package -Dexample.project.dir=/path/to/thorntail-rest-http
 ```
 
-## booster.project.dir
+## `example.project.dir`
 
-Path to the booster project directory, e.g.:
+Path to the example project directory, e.g.:
 
 ```bash
--Dbooster.project.dir=/path/to/thorntail-rest-http
+-Dexample.project.dir=/path/to/thorntail-rest-http
 ```
 
-NOTE: The dir content is copied to `target/generated-project`. `src/test/` and `target` subdirs are skipped.
+## `external.license.service` (optional)
 
-## booster.name
-
-The name used in licenses.html report, e.g.:
-
-```bash
--Dbooster.name="Thorntail - REST API Level 0"
-```
-
-## booster.version
-
-The version used in licenses.html report, e.g.:
-
-```bash
--Dbooster.version=2.4.0-redhat-1
-```
-
-## booster.assembly.name
-
-The name of the generated archive, e.g.:
-
-```bash
--Dbooster.assembly.name=thorntail-rest-http
-```
-
-results in `target/thorntail-rest-http-licenses.zip`.
-
-## booster.product.build
-
-Boolean indicating whether this is a product variant of the booster or not, e.g.:
+URL of an external license service that can provide license information.
+If not set, no external license service is used.
 
 ```bash
--Dbooster.product.build=true
+-Dexternal.license.service=http://.../find-license-check-record-and-license-info
 ```
 
-## booster.repo.url
+## product variant of the example?
```

Comment:
Is the `?` mark intentional ?

Reply:
It's meant to be a question, which is later answered, but I agree it can be confusing. I'll rephrase.